content/docs/querying/operators.md

```diff
@@ -17,7 +23,7 @@ The following binary arithmetic operators exist in Prometheus:
 Binary arithmetic operators are defined between scalar/scalar, vector/scalar,
 and vector/vector value pairs.
 
-**Between two scalars**, the behavior is obvious: they evaluate to another
+**Between two scalars**, the behaviour is obvious: they evaluate to another
```

Comment:
Oh, I didn't know that rule. Actually, I'm not used to seeing commas after "i.e."/"e.g." - do you perhaps have a link pointing at rules? If that's the way, I'm fine with it.

Though with the British/US thing I'm mostly after the spelling of words. I didn't even know there were comma differences between US/British, except for the Oxford comma, which I think really adds clarity, but can also be used in US English.